packages/ios-client/Sources/FishjamClient/media/Dimensions.swift

```diff
@@ -1,23 +1,31 @@
 import CoreMedia
 import Foundation
 
-/// Type refering to video dimensions.
-public typealias Dimensions = CMVideoDimensions
-
-extension Dimensions {
+public struct Dimensions {
     public static let aspect16By9 = 16.0 / 9.0
     public static let aspect4By3 = 4.0 / 3.0
+
+    private var dimensions: CMVideoDimensions
+
+    public var width: Int32 { dimensions.width }
+    public var height: Int32 { dimensions.height }
+
+    public init(width: Int32, height: Int32) {
+        self.dimensions = CMVideoDimensions(width: width, height: height)
+    }
+
+    public init(_ dimensions: CMVideoDimensions) {
+        self.dimensions = dimensions
+    }
+
+    /// Swaps height with width.
```

Comment:
Unfortunately I can't modify the `dimensions` properties because it's immutable. I left it as it was because earlier it also returned new struct. But I'll think about a better wording for that

Reply:
Sth like `returns struct with flipped dimensions` should be enough.  Or you can use mutating function to do swap.

Reply:
I've already commited updated wording but thanks for the suggestion!